com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs

```diff
@@ -125,6 +132,14 @@ internal bool IsSceneAddressable(string sceneName)
             return false;
         }
 
+        internal void RemoveAddressablesPreferences()
+        {
+#if UNITY_EDITOR
```

Comment:
It would be great to add a comment right before `#if UNITY_EDITOR` explaining why we have an empty runtime part of this method.
Something like _This is Editor-only code. Runtime part of this method should be empty. _ would be enough.